internal/jobs/twitter/keytype.go

```diff
@@ -0,0 +1,41 @@
+package twitter
+
+import (
+	"fmt"
+	"strings"
+	"github.com/masa-finance/tee-worker/pkg/client"
+)
+
+// DetectTwitterKeyType tries to determine if the key is base, elevated, or credential.
+func DetectTwitterKeyType(apiKey string) (string, error) {
```

Comment:
Can we have a `TwitterKeyType` type instead of just returning `string`? With values `elevated`, `base` and `unknown`.